lucid/overview.py

```diff
@@ -354,3 +361,74 @@ def add_from_dict(self, items):
         for location, dev_list in items.items():
             stand, system = location.split("|")
             self.add_devices(dev_list, stand=stand, system=system)
+
+
+class QuickAccessToolbar(QtWidgets.QWidget):
+    """Tab Widget with tabs containing buttons defined via a yaml file"""
+    def __init__(self, parent=None):
+        super().__init__(parent=parent)
+        self._tools = None
+        self.setSizePolicy(QtWidgets.QSizePolicy.Minimum,
+                           QtWidgets.QSizePolicy.Minimum)
+        self._setup_ui()
+
+    def sizeHint(self):
+        return QtCore.QSize(100, 100)
+
+    @Property(str)
+    def toolsFile(self):
+        return self._tools_file
+
+    @toolsFile.setter
+    def toolsFile(self, file):
+        if not file:
+            return
+        if isinstance(file, (str, bytes, os.PathLike)):
+            with open(self._tools_file, 'r') as tf:
+                self._tools = yaml.full_load(tf)
+        else:
+            self._tools = yaml.full_load(file)
+        self._assemble_tabs()
+
+    def _setup_ui(self):
+        main_layout = QtWidgets.QVBoxLayout()
+        self.setLayout(main_layout)
+        self.tab = QtWidgets.QTabWidget()
+        main_layout.addWidget(self.tab)
+
+    def _assemble_tabs(self):
+        self.tab.clear()
+        try:
+            for tab_name, tab_items in self._tools.items():
+                page = QtWidgets.QWidget()
+                page.setLayout(SnakeLayout(4))
```

Comment:
Number of columns should be exposed out of here

Reply:
It is now exposed at the yaml file definition under `config`